app/views/assessments/show.html.erb

```diff
@@ -65,7 +65,7 @@
     <% if @cud.course_assistant? or @cud.instructor? then %>
         <h3>Admin Options (CA)</h3>
         <ul class="options">
-          <li><%= link_to "Grade section #{@cud.section} submissions", url_for(:action => 'viewGradesheet', :section => '1', :escape => false), 
+          <li><%= link_to "Grade section #{@cud.section.downcase} submissions", url_for(:action => 'viewGradesheet', :section => '1', :escape => false), 
```

Comment:
We want to keep `@cud.secion` uppercase, actually. This is usually something that people will expect to be uppercase, like a single letter (Section H), or more generally some name (think: proper noun) for the section.